rules/windows/credential_access_relay_ntlm_auth_via_http_spoolss.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2022/04/30"
+maturity = "production"
+updated_date = "2022/04/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempt to coerce a local NTLM authentication via HTTP using the Windows Printer Spooler service as a target.
+An adversary may use this primitive in combination with other techniques to elevate privileges on a compromised system.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Local NTLM Relay via HTTP"
+references = [
+    "https://github.com/med0x2e/NTLMRelay2Self",
+    "https://github.com/topotam/PetitPotam",
+    "https://github.com/dirkjanm/krbrelayx/blob/master/printerbug.py",
+]
+risk_score = 73
+rule_id = "4682fd2c-cfae-47ed-a543-9bed37657aa6"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+  process.name : "rundll32.exe" and
+  /* relay to HTTP via spoolss or srvsvc pipes */
+  process.command_line : ("*davclnt.dll*http*spoolss*", "*davclnt.dll*http*pipe*srvsvc", "*davclnt.dll*http*/print/PIPE/*")
```

Comment:
Is `"*davclnt.dll*http*pipe*srvsvc"` needed? I didn't see anything in your screenshot that matched on this.

Reply:
![image](https://user-images.githubusercontent.com/64742097/167201679-e1e55f2d-6b2e-41ce-916b-b085eb6ac58a.png)

search was KQL (case sens, that's why it didn't show up)